src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -2,6 +2,7 @@
 from __future__ import absolute_import
 
 import re
+from distutils.util import strtobool
```

Comment:
Can you use `tools.str_to_bool` instead?

Reply:
I think @votdev has been advocating`distutils.util` in another PR before. I don't mind switching to `tools` instead, but it would be helpful to have this unified throughout the project then.